pkg/reconciler/mtbroker/broker.go

```diff
@@ -255,13 +230,6 @@ func (r *Reconciler) getChannelTemplate(ctx context.Context, b *eventingv1.Broke
 	}, nil
 }
 
-func (r *Reconciler) FinalizeKind(ctx context.Context, b *eventingv1.Broker) pkgreconciler.Event {
```

Comment:
hmmmm This will cause us to not be able to delete brokers without manually removing the finalizer. I am pretty sure the genreconciler does not remove finalizers if `FinalizeKind` is not implemented.

Reply:
Looks to work just fine?

```
vaikas-a01:eventing vaikas$ kubectl -n vaikas-test get brokers
NAME       READY   REASON   URL                                                                             AGE
broker2    True             http://broker-ingress.knative-eventing.svc.cluster.local/vaikas-test/broker2    5d19h
broker99   True             http://broker-ingress.knative-eventing.svc.cluster.local/vaikas-test/broker99   5m16s
vaikas-a01:eventing vaikas$ kubectl -n vaikas-test delete brokers  broker99
broker.eventing.knative.dev "broker99" deleted
vaikas-a01:eventing vaikas$ kubectl -n vaikas-test get brokers
NAME      READY   REASON   URL                                                                            AGE
broker2   True             http://broker-ingress.knative-eventing.svc.cluster.local/vaikas-test/broker2   5d19h
```